stripe/res/values/strings.xml

```diff
@@ -187,4 +187,5 @@
     <!-- Failure Reason -->
     <string name="stripe_failure_reason_authentication">We are unable to authenticate your payment method. Please choose a different payment method and try again.</string>
     <string name="stripe_failure_reason_timed_out">Timed out authenticating your payment method -- try again</string>
+    <string name="stripe_failure_connection_error">There was an unexpected error -- try again in a few seconds</string>
```

Comment:
We will need to run through localization again.  Perhaps for Public Beta

Reply:
I think it's helpful to say that this is a connection error. The user might be able to diagnose the issue by checking if they have an internet connection.
This is from the [web demo](https://stripe-payments-demo.appspot.com/):
<img width="1666" alt="Screen Shot 2021-04-06 at 10 41 00 AM" src="https://user-images.githubusercontent.com/77990083/113755031-b2a75800-96c4-11eb-9f9a-c9ef766e9114.png">

I also would add a "please" [...try again...]